weave

```diff
@@ -48,7 +48,7 @@ weave launch        [--password <pass>] [--trusted-subnets <cidr>,...]
                     [--no-restart] [--resume] [--no-discovery] [--no-dns]
                     [--ipalloc-init <mode>]
                     [--ipalloc-range <cidr> [--ipalloc-default-subnet <cidr>]]
-                    [--plugin]
+                    [--plugin=false --proxy=false]
```

Comment:
OIC, I previously thought you were referring to the values and overlooked the brackets grouping both options. Fixed in latest version.